plugins/Toolbox/src/Toolbox.py

```diff
@@ -325,6 +346,9 @@ def getRemotePackage(self, package_id: str) -> Optional[Dict]:
     # --------------------------------------------------------------------------
     @pyqtSlot(str, result = bool)
     def canUpdate(self, package_id: str) -> bool:
+        if self._isOldPlugin(package_id):
```

Comment:
I assume from this that regardless of version, we want people to update plugins from the old plugin browser to the new packages?

Reply:
yes indeed